commit 6ae4bb6612c675d4132bd4fcc7cac4eef46f9624
Author: Christian Lamparter <chunkeey@googlemail.com>
Date:   Fri Jan 13 16:35:05 2012 +0100

    carl9170: remove eeprom data injection option
    
    In the early days, this was a quite useful software
    feature for testing different regdomains and chain
    configurations without adding debugfs cruft into
    the driver. Nowadays, the driver's phy code seems
    to be stable and there's no need for it anymore.
    Therefore I decided to removed altogether.
    
    Signed-off-by: Christian Lamparter <chunkeey@googlemail.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/ath/carl9170/main.c b/drivers/net/wireless/ath/carl9170/main.c
index 4dfb0290b43a..6ee7c55f75a8 100644
--- a/drivers/net/wireless/ath/carl9170/main.c
+++ b/drivers/net/wireless/ath/carl9170/main.c
@@ -1934,10 +1934,6 @@ int carl9170_register(struct ar9170 *ar)
 	if (err)
 		return err;
 
-	err = carl9170_fw_fix_eeprom(ar);
-	if (err)
-		return err;
-
 	err = carl9170_parse_eeprom(ar);
 	if (err)
 		return err;